bitcoin/src/blockdata/transaction.rs

```diff
@@ -642,7 +646,7 @@ impl Transaction {
     /// have the information to determine.
     /// - Does NOT handle the sighash single bug (see "Return type" section)
     ///
-    /// # Return type
+    /// # Returns
```

Comment:
Seems correct to me to use `Returns` since the section describes the `EncodeSigningDataResult` return type not just the error path? Am I missing something?

(Did catch the incorrect section name in line 643/647 above though thanks to this comment :)

Reply:
Oh, yes, the return type is not `Result` :man_facepalming: